calcolatrice.py

```diff
@@ -42,8 +42,9 @@ def exit_program():
         valore_finale = check_finale("sommare")
         check_null(valore_finale)
 
-        risultato = valore_iniziale + valore_finale
-        print(f"Il risultato della somma è: ", risultato)
+        if (valore_iniziale.isnumeric() & valore_finale.isnumeric()):
+            risultato = int(valore_iniziale) + int(valore_finale)
```

Comment:
Proprio perche' gli input sono stringhe la somma era "sbagliata". Senza il cast a int (`int()`) il programma faceva `"1"` + `"1"`, che risulta in una concatenazione, ovvero `"11"` :)